client/components/happiness-support/test/index.jsx

```diff
@@ -81,137 +96,129 @@ describe( 'HappinessSupport', () => {
 			recordTracksEvent: noop,
 		};
 
-		beforeEach( () => {
-			wrapper = shallow(
+		test( 'should be rendered only when showLiveChatButton prop is true and LiveChat is available', () => {
+			const { rerender } = render(
 				<HappinessSupport { ...props } showLiveChatButton={ true } liveChatAvailable={ true } />
```

Comment:
There's at least a few more of those we could cleanup:

```suggestion
				<HappinessSupport { ...props } showLiveChatButton liveChatAvailable />
```